<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameModeWindow</class>
 <widget class="QWidget" name="GameModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выбор режима</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButtonPlayer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-image: url(:/images/images/SS_PB_GMW_Player.png);
    }
QPushButton:hover {
	background-image: url(:/images/images/SS_PB_GMW_Player_hover.png);
}
QPushButton:pressed {
	background-image: url(:/images/images/SS_PB_GMW_Player_push.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMaster">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-image: url(:/images/images/SS_PB_GMW_Master.png);
    }
QPushButton:hover {
	background-image: url(:/images/images/SS_PB_GMW_Master_hover.png);
}
QPushButton:pressed {
	background-image: url(:/images/images/SS_PB_GMW_Master_push.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtBack">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>310</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;
        border: none;
    }
QPushButton:hover {
	image: url(:/images/images/SS_PB_MM_Exit.png);
}
QPushButton:pressed {
	image: url(:/images/images/SS_PB_MM_Exit.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {background-image: url(:/images/images/SS_L_MM.png);}</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>pushButtonPlayer</zorder>
  <zorder>pushButtonMaster</zorder>
  <zorder>pushButtBack</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
